packages/editor-ui/src/components/TagsContainer.vue

```diff
@@ -70,7 +70,22 @@ interface TagEl extends ITag {
 export default defineComponent({
 	name: 'TagsContainer',
 	components: { IntersectionObserver, IntersectionObserved },
-	props: ['tagIds', 'limit', 'clickable', 'responsive', 'hoverable'],
+	props: {
+		tagIds: {
+			type: Array as () => string[],
+			required: true,
+		},
+		limit: {
+			type: Number,
+			default: null,
```

Comment:
Nit: Should we expand the type to `Number | null` or should the default here be `DEFAULT_MAX_TAGS_LIMIT`?

Reply:
Good point. `DEFAULT_MAX_TAGS_LIMIT` makes more sense here